sections/sections-app/src/java/org/sakaiproject/tool/section/jsf/backingbean/CourseDependentBean.java

```diff
@@ -156,5 +157,26 @@ public PreferencesBean getPrefs() {
 	public String getSiteRole() {
 		return getCourseBean().authz.getRoleDescription(getUserUid(), getSiteContext());
 	}
+	protected boolean canManageAnySection() {
+		if (getCourseBean().authz.getRoleDescription(getUserUid(), getSiteContext())
+				.equals(JsfUtil.getLocalizedMessage("admin_role")))
```

Comment:
I don't understand this line.

The common call in Sakai is:

securityService().isSuperUser()

Reply:
Yes, I did it the same way as it was in "AuthzSakaiImpl.getRoleDescription(String userUid, String siteContext)".
But you are right, it is better to fix it and not continue using the Cover of the securityService